src/test/style.ts

```diff
@@ -1,16 +1,15 @@
-var assert = require('assert');
-var fakeRaf = require('fake-raf');
+import assert from 'assert'
 
-var snabbdom = require('../snabbdom');
-fakeRaf.use();
```

Comment:
When using `require` calls, with whatever karma uses as a module system, this `.use()` call occurs before the style module is evaluated. But converting this to TypeScript and static imports, the only way to have the `.use()` call occur before the import, seems to be using dynamic `import()` call. If we had top-level `await` then it would have been reasonable. But [TypeScript doesn't support that yet](https://github.com/microsoft/TypeScript/issues/25988).

So I removed this sweet little tool by @paldepind and replaced it with actual calls to the real `requestAnimationFrame`. See the test below.